PrebidMobile/AdUnits/AdUnit.swift

```diff
@@ -74,6 +74,10 @@ public class AdUnit: NSObject, DispatcherDelegate {
         // PBS should cache the bid for original api.
         Prebid.shared.useCacheForReportingWithRenderingAPI = true
     }
+    
+    deinit {
+        dispatcher?.invalidate()
```

Comment:
Can we invalidate / stop dispatcher in the `deinit` of `Dispatcher` itself?

Reply:
Unfortunately, we can't cause `Timer` holds a strong reference to `Dispatcher`, which means `deinit` is never called until `Timer` gets invalidated.